app/js/controllers.js

```diff
@@ -4415,7 +4416,7 @@ angular.module('myApp.controllers', ['myApp.i18n'])
       })
     }
 
-    Storage.get('notify_nodesktop', 'send_ctrlenter', 'notify_volume', 'notify_novibrate', 'notify_nopreview', 'notify_nopush').then(function (settings) {
+    Storage.get('notify_nodesktop', 'send_ctrlenter', 'notify_volume', 'notify_novibrate', 'notify_nopreview', 'notify_nopush', 'appearance_nightmode').then(function (settings) {
```

Comment:
We should be referring to the theme as 'dark' not 'night'

Reply:
done